terraform.tfvars.example

```diff
@@ -0,0 +1,7 @@
+env=""
+project_id=""
+gcp_billing_account="010C5C-D2E6F2-FA1F92"
+gcp_folder_id="979069919925"
+stackdriver_workspace=""
```

Comment:
Did we keep `stackdriver_workspace` on purpose?. If the plan is the workspace is the same as the project_id, then this can be removed.

Reply:
Yep - purposeful - the project id and workspace id may not always be the same.